cmssw-tool-conf.spec

```diff
@@ -5,190 +5,22 @@
-Requires: pyquen-toolfile
-Requires: pythia6-toolfile
-Requires: pythia8-toolfile
-Requires: python-toolfile
-Requires: python3-toolfile
-Requires: root-toolfile
-Requires: sherpa-toolfile
-Requires: libpciaccess-toolfile
-Requires: numactl-toolfile
-Requires: hwloc-toolfile
-%ifnarch aarch64
-Requires: gdrcopy-toolfile
-%endif
-Requires: ucx-toolfile
-Requires: openmpi-toolfile
-Requires: sigcpp-toolfile
-Requires: sqlite-toolfile
-Requires: systemtools
-Requires: tauola-toolfile
-Requires: tauolapp-toolfile
-Requires: thepeg-toolfile
-Requires: toprex-toolfile
-Requires: libuuid-toolfile
-Requires: xerces-c-toolfile
-Requires: zlib-toolfile
-Requires: dcap-toolfile
-Requires: frontier_client-toolfile
-Requires: xrootd-toolfile
-Requires: dd4hep-toolfile
-Requires: graphviz-toolfile
-Requires: valgrind-toolfile
-Requires: cmsswdata-toolfile
-Requires: zstd-toolfile
-Requires: hls-toolfile
-Requires: opencv-toolfile
-Requires: grpc-toolfile
-Requires: onnxruntime-toolfile
-Requires: triton-inference-server-toolfile
-Requires: hdf5-toolfile
-Requires: rivet-toolfile
-Requires: cascade-toolfile
-Requires: yoda-toolfile
-Requires: fftw3-toolfile
-Requires: fftjet-toolfile
-Requires: professor2-toolfile
-Requires: xz-toolfile
-Requires: lz4-toolfile
-Requires: protobuf-toolfile
-Requires: lcov-toolfile
-Requires: llvm-gcc-toolfile
-Requires: tbb-toolfile
-Requires: mctester-toolfile
-Requires: vdt-toolfile
-Requires: icc-gcc-toolfile
-Requires: gnuplot-toolfile
-Requires: sloccount-toolfile
-Requires: millepede-toolfile
-Requires: pacparser-toolfile
-Requires: git-toolfile
-Requires: cgal-toolfile
-Requires: doxygen-toolfile
-Requires: yaml-cpp-toolfile
-Requires: gmp-static-toolfile
-Requires: mpfr-static-toolfile
-Requires: fastjet-contrib-toolfile
-Requires: opencl-toolfile
-Requires: opencl-cpp-toolfile
-Requires: qd-toolfile
-Requires: blackhat-toolfile
-Requires: sherpa-toolfile
-Requires: geant4-parfullcms-toolfile
-Requires: fasthadd
-Requires: eigen-toolfile
-Requires: gdb-toolfile
-Requires: libxslt-toolfile
-Requires: giflib-toolfile
-Requires: freetype-toolfile
-Requires: utm-toolfile
-Requires: libffi-toolfile
-Requires: CSCTrackFinderEmulation-toolfile
-Requires: tinyxml2-toolfile
-Requires: md5-toolfile
-Requires: gosamcontrib-toolfile
-Requires: gosam-toolfile
-Requires: madgraph5amcatnlo-toolfile
-Requires: python_tools
-Requires: dasgoclient
-Requires: OpenBLAS-toolfile
-Requires: mxnet-predict-toolfile
-Requires: mkfit-toolfile
-Requires: dablooms-toolfile
-
-# Only for Linux platform.
-%ifos linux
-Requires: openldap-toolfile
-Requires: gperftools-toolfile
-Requires: cuda-toolfile
-Requires: alpaka-toolfile
-Requires: cupla-toolfile
-
-%if "%{cmsos}" != "slc7_aarch64"
-Requires: cudnn-toolfile
-%endif
-
-Requires: libunwind-toolfile
-%ifnarch ppc64le
-Requires: igprof-toolfile
-Requires: openloops-toolfile
-%endif
-
-%ifarch x86_64
-Requires: dmtcp-toolfile
-Requires: tkonlinesw-toolfile
-Requires: oracle-toolfile
```

Comment:
@mrodozov , we need to keep the dependencies but should replace `<tool>-toolfile` with `<tool>` only.
This should also be done for coral and fwlite tool-conf.spec

Reply:
this file is containing a short list because so I first finish pkgs with any script logic, the short list if for not deploying all xmls every time (not to build the whole cmssw-tool-conf)

Reply:
this type of changes are better to test locally as this will help with debugging.